src/lib/SapphireClient.ts

```diff
@@ -201,9 +212,30 @@ declare module 'discord.js' {
 	interface ClientOptions {
 		id?: string;
 		logger?: ClientLoggerOptions;
+		i18n?: ClientInternationalizationOptions;
 	}
 
 	interface ClientLoggerOptions {
-		level: LogLevel;
+		level?: LogLevel;
+		instance?: ILogger;
+	}
+
+	interface ClientInternationalizationOptions {
+		defaultName?: string;
+		instance?: IInternationalization;
+	}
+
+	interface Message {
+		fetchLanguage(): Awaited<string>;
+		fetchLanguageKey(key: string, ...values: readonly unknown[]): Promise<string>;
```

Comment:
In i18n for instance keys are defined as `Record<string, string>`.
Could that be a possible overwrite?